notebooks/data-sources/TestGrid/metrics/probability_to_fail.ipynb

```diff
@@ -0,0 +1,1091 @@
+{
```

Comment:
<p>Can you re-iterate the problem statement here? as in what are you trying to calculate and why did we choose Calibrated Classification model to be a suitable approach here as opposed to any other standard classification technique? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/310/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>+1</p>

Reply:
<p>Sure. We will add the explanation.</p>